### DISCUSSION TITLE
Item description of Magnetic Suspension Crate is misleading

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14396] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It says "While powered, it guarantees the contents do not suffer from impact damage."
but in fact it only prevents Volatile Nitroglycerin from decreasing condition over time

### Reproduction steps

1. spawn a Magnetic Suspension Crate
2. keep the crate powered and put Nitroglycerin inside 
3. drop it from a high place
4. explode

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
This will be addressed in the next update, thanks for your report!
